### PR TITLE
Hotfix: sanitizer config type definition

### PR DESCRIPTION
**Pull request by phanletrunghieu**
When I use sanitizer with typescript.

I wanna use a function with `el` param.

```js
const sanitizerConfig = {
  b: true,
  p: function(el) {
    return !el.textContent.includes('bad text')
  }
}
```

(https://editorjs.io/sanitizer)

But sanitizer is definited with `func(): any` (missing `el` parram).

### CODE REVIEW

**Review by neSpecc — CHANGES_REQUESTED** [review]
No description provided.

File: types/configs/sanitizer-config.d.ts [hunk 1]
```diff
@@ -31,5 +31,5 @@ export interface SanitizerConfig {
    *   }
    * }
    */
-  [key: string]: boolean|{[attr: string]: boolean|string}|(() => any);
+  [key: string]: boolean|{[attr: string]: boolean|string}|((el?: any) => any);
```
Comment (neSpecc) [inline types/configs/sanitizer-config.d.ts:34]: why do you use `any` type instead of `Element`?

**Review by neSpecc — CHANGES_REQUESTED** [review]

**Comment by hata6502** [comment]
https://github.com/codex-team/editor.js/pull/1456

**Comment by khaydarov** [comment]
@hata6502 could you resolve Peters' comment please?

File: types/configs/sanitizer-config.d.ts (hunk 1 above)
Comment (phanletrunghieu) [reply types/configs/sanitizer-config.d.ts]: Reasonable

**Review by neSpecc — APPROVED** [review]

**Review by khaydarov — APPROVED** [review]

**Pull request merged**